lib/ansible/plugins/connection/psrp.py

```diff
@@ -30,6 +30,13 @@
     vars:
     - name: ansible_user
     - name: ansible_psrp_user
+  password:
```

Comment:
version_added? or this was just missing?

Reply:
Technically `ansible_password` has worked historically (just through play context) and because I was planning on backporting this I didn't think adding `version_added` would be ideal.